src/styles/blocks/card.scss

```
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: 198px;
+  height: 406px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: var(--white-accent);
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+}
+
+.product-card__top {
+  padding: 32px 19px 40px;
+}
+
+.product-card__image {
```

1. make image as a block element 
2. place at the center  
3. add outer indent
```suggestion
.product-card__image {
  display: block;
  margin: 0 auto 40px;
```